hudi-common/src/main/java/org/apache/hudi/common/table/timeline/HoodieInstantTimeGenerator.java

```diff
@@ -156,7 +156,7 @@ public static String getInstantFromTemporalAccessor(TemporalAccessor temporalAcc
     return MILLIS_INSTANT_TIME_FORMATTER.format(temporalAccessor);
   }
 
-  public static String getCurrentInstantTimeStr() {
+  public static String getCurrentTimeAsString() {
```

Comment:
rename: currentInstantTimeString() 

I prefer to keep the intent of using this as instant time - with the name.. "CurrentTime" makes it less obvious at call sites.